File: Modul-6/Soal-3/PRAK603-2410817220012-NOOR-KHALISA.py
n1, n2 = map(int, input("n1 dan n2: ").split())
if n1 != n2:
    print("Jumlah tidak sama.")
else:
    baris1 = list(map(int, input("Baris pertama: ").split()))
    baris2 = list(map(int, input("Baris kedua  : ").split()))
    hasil = [baris1[i] * baris2[i] for i in range(n1)]
    print(" ".join(map(str, hasil)))

File: Modul-6/Soal-3/PRAK603-2410817220012-NOOR-KHALISA.c
#include <stdio.h>
int main(){
    int n1, n2;
    printf("n1 dan n2: ");
    scanf("%d %d", &n1, &n2);

    if (n1 != n2)
    {
        printf("Jumlah tidak sama.");
    }else
    {
        int baris1[n1], baris2[n2], hasil[n1];
        for (int i = 0; i < n1; i++)
        {
            scanf("%d", &baris1[i]);
        }
        for (int i = 0; i < n2; i++)
        {
            scanf("%d", &baris2[i]);
        }
        for (int i = 0; i < n1; i++)
        {
            hasil[i] = baris1[i] * baris2[i];
        }
        for (int i = 0; i < n1; i++)
        {
            printf("%d ", hasil[i]);
        } 
        printf("\n");
    }
    return 0;
}